x-pack/plugins/uptime/server/lib/adapters/telemetry/kibana_telemetry_adapter.ts

```diff
@@ -194,15 +192,14 @@ export class KibanaTelemetryAdapter {
       },
     };
 
-    const { body: result } =
-      typeof callCluster === 'function'
-        ? await callCluster('search', params)
-        : await callCluster.search(params);
+    const { body: result } = isUptimeESClient(callCluster)
+      ? await callCluster.search(params)
+      : await callCluster.search<ESSearchResponse<unknown, typeof params>>(params);
```

Comment:
with your last commit i think, we can simplify this.

Reply:
Great thinking, thank you for pointing that out!
I've replaced the conditional [here](https://github.com/elastic/kibana/pull/86718/commits/f63fe37a28464ef58583c71a0bc68d2cb26b0969#diff-b14707a96d135f28f2326ccb972bcf2b15514928e2c73d9627e3ea79c29001d8R190)